rocketchatctl

```diff
@@ -777,7 +777,7 @@ rocketchat_health_check(){
     local count=0
     while ! ([ $rocket_healthy -eq  1 ] || [ $count -eq 150 ]); do
         get_rocketchat_current_version
-        [[ $current_rocketchat_version =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] && rocket_healthy=1
+        [[ $current_rocketchat_version =~ ^[0-9]+\.[0-9]+$ ]] && rocket_healthy=1
```

Comment:
My Regex-fu is not strong right now.  But we should make that last one optional to keep it from breaking < 4.0.0 releases

Reply:
ahhhh true.. I didn't realize it is checking beginning and ending chars.. I'll adjust accordingly.